packages/flutter_tools/lib/src/reporting/unified_analytics.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:unified_analytics/unified_analytics.dart';
+
+import '../globals.dart' as globals;
+import '../version.dart';
+
```

Comment:
The logic for this function is the same as to what was used for package:usage.

https://github.com/eliasyishak/flutter/blob/use-same-analytics-logic-as-pkg-usage/packages/flutter_tools/lib/src/reporting/usage.dart#L152-L163